### ISSUE TITLE
classifier

### ISSUE THREAD

**Issue by geraldstanje** [cyhex/streamcrab#20]
Hi,

Do you want to compare your classifier with other teams competed at semeval2014 and tell me the score?
http://alt.qcri.org/semeval2014/task9/
paper and results: http://alt.qcri.org/semeval2014/cdrom/pdf/SemEval2014009.pdf

Thanks,
Gerald

**Comment by cyhex** [comment]
well they had 2 subtasks and they used same Datasets as 2013 task2, I did compared several classifiers with GoldData from task2 but i did not used training set from semeval. 
Results: https://github.com/cyhex/streamcrab/blob/master/docs/acurracy_tests.md
If you could point me to he training dataset from semeval2014 i could provide test results

**Comment by geraldstanje** [comment]
i tried to send you an email, is your mailbox full?

**Comment by cyhex** [comment]
no but it limits the attachment's size.
perhaps you could upload it to some kind of dropbox ?
Thanks,

**Comment by geraldstanje** [comment]
please give me another email address.

**Comment by cyhex** [comment]
I trained the classifier using 2014_a_train.txt and tested against 2014_a_test_gold.txt.
Final score is: 0.711
testing 2014_a_test_gold.txt but just Twitter2013/Twitter2014 labels produces: 0.723

**Comment by cyhex** [comment]
One more thing, my system classifies only with positive/negative, Objective (neutral) entities are ignored

**Comment by geraldstanje** [comment]
what about task b?

2014-08-05 15:19 GMT+02:00 cyhex notifications@github.com:

**Comment by cyhex** [comment]
Task B: 0.770

**Comment by geraldstanje** [comment]
did you only train your classifier with the train dataset?

2014-08-05 15:38 GMT+02:00 cyhex notifications@github.com:

**Comment by cyhex** [comment]
yes for test B with training B set and A with training A set.

**Comment by geraldstanje** [comment]
can you also add the neutral class? just curious how the performance
changes...

2014-08-05 16:57 GMT+02:00 cyhex notifications@github.com:

**Comment by cyhex** [comment]
i am not sure how much work would be needed as i build the system to be bi polar only.

**Comment by cyhex** [comment]
but i believe it is not going to change much as in case of B test the positive / negative tests passed with 77%, ignoring the Objective tests. Which is consistent with SemEval 2013 best tests in witch B test was 78.5%

**Comment by geraldstanje** [comment]
how can you come up with a test for b if you cant classify neutral class?

2014-08-05 17:12 GMT+02:00 cyhex notifications@github.com:

**Comment by geraldstanje** [comment]
idk where u got 78.5% from....you should also look a the right paper:
http://www.aclweb.org/anthology/S/S13/S13-2052.pdf

2014-08-05 17:15 GMT+02:00 Gerald Stan gerald.stanje@gmail.com:

**Comment by cyhex** [comment]
I think there is a misunderstanding, my scores are not direct comparable with SemEval scores. I only provide accuracy results of positive and negative classified text. The tests are against Gold data from SemEval13/14. 
Just to clarify, a score of 1 (100%) would mean that my classifier identified correctly all Positives and Negatives in Gold test dataset.

**Comment by geraldstanje** [comment]
ok.

2014-08-05 18:14 GMT+02:00 cyhex notifications@github.com:
